### ISSUE TITLE
Add access permissions to document root (_cci1/language-php.md)

### ISSUE THREAD

**Issue by tms320c** [circleci/circleci-docs#655]
The sample of Apache configuration is not correct. It is mandatory to add access permissions to the document root:

`<Directory /home/ubuntu/MY-PROJECT/server-root>`
`    Options Indexes FollowSymLinks`
`    AllowOverride All`
`    Require all granted`
` </Directory>`

and change Apache user to ubuntu (in /etc/apache2/envvars : 

`export APACHE_RUN_USER=ubuntu`
`export APACHE_RUN_GROUP=ubuntu`

Without these changes the sample configuration does not work.

In addition, it may be necessary to enable rewrite mod (add `sudo a2enmod rewrite` to circle.yml) and add `ServerName host.example.com` somewhere in the Apache configuration.

If  PHP 7 is required then it is necessary to disable php5 module (`sudo a2dismod php5`) and change `LoadModule` directive parameters for something different. For example, for PHP 7.0.7:
`LoadModule php7_module /opt/circleci/php/7.0.7/usr/lib/apache2/modules/libphp7.so`

**Comment by pankaj9296** [comment]
It should be included in the documentation.

**Comment by ryanwohara** [comment]
The docs reflect Apache 2.2 instead of Apache 2.4. I doubt anyone will be using Apache 2.2 in CircleCI 2.0. This is definitely worth fixing.

**Comment by smart-alek** [comment]
#1232 and #1233 almost certainly related.

**Comment by felicianotech** [comment]
@smart-alek CircleCI 2.0 is the "fix".

**Comment by smart-alek** [comment]
@tms320c and @pankaj9296 

Thanks for the feedback, and apologies for responding so late! Have either of you tried CircleCI 2.0? I'm trying to figure out if this issue has followed us into 2.0, or if it was resolved when we moved versions.

With our 1.0 EOL happening in August, it's getting increasingly hard to justify changes to 1.0 docs, so I will be very happy if moving to 2.0 fixed things for you. Let us know!
